### DISCUSSION TITLE
Is it correct that there are the same files in both paths?

### DISCUSSION THREAD

**Discussion by Doddddd** [waydroid/waydroid#612] [Q&A]
Is it correct that there are the same files in both paths?
/var/lib/waydroid/data
~/.local/share/waydroid/data

**Comment by cniw** [comment]
yes, because it's just mounted with bind option from that folder
you can see it on you log after you start waydroid
`sudo grep data$ /var/lib/waydroid/waydroid.log`
here is the code:
* https://github.com/waydroid/waydroid/blob/5159f6583109f402e3b51c528c81219bb6821110/tools/actions/container_manager.py#L101-L103
* https://github.com/waydroid/waydroid/blob/cca8af2d7d3494d887d9655f8e78244b5a2ca513/tools/config/__init__.py#L72-L73
* https://github.com/waydroid/waydroid/blob/cca8af2d7d3494d887d9655f8e78244b5a2ca513/tools/config/__init__.py#L55
